install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -11,18 +11,22 @@ metadata:
     release: {{ .Release.Name }}
 data:
   mesh: |-
+    {{- if .Values.mixer.policy.enabled }}
     # Set the following variable to true to disable policy checks by the Mixer.
     # Note that metrics will still be reported to the Mixer.
     disablePolicyChecks: {{ .Values.global.disablePolicyChecks }}
+    {{- end }}
 
+    {{- if .Values.mixer.telemetry.enabled }}
```

Comment:
i don't think this is correct. it is possible to have tracing without `istio-telemetry` and this flag is for envoys.

Reply:
hmm, that's right. I'll update.